pycbc/events/coinc.py

```diff
@@ -40,7 +40,10 @@
 _APPROXIMANT_DURATION_MAP = {
     'SEOBNRv2duration': 'SEOBNRv2',
     'SEOBNRv4duration': 'SEOBNRv4',
-    'SEOBNRv5duration': 'SEOBNRv5_ROM'
+    'SEOBNRv5duration': 'SEOBNRv5_ROM',
+    'IMRPhenomXASduration': 'IMRPhenomXAS',
+    'TaylorF2duration': 'TaylorF2',
+    'SPAtmpltduration': 'SPAtmplt',
```

Comment:
Do we still need this dictionary?? Especially given #4540

Reply:
Hopefully not!

Reply:
Yup. Closing this!